src/test/regress/expected/custom_citus_test.out

```diff
@@ -0,0 +1,22 @@
+CREATE TABLE dist(a int, b int);
```

Comment:
Very promising! 

Can we maybe have a create table file separated, and then run the tests? So that we can use MX workers to run the queries?

Reply:
I guess we do not need this file anymore?